src/components/donation/Steps/Submit/Crypto/TxSubmit/estimateDonation.ts

```diff
@@ -14,11 +14,14 @@ import { tokenBalance } from "./tokenBalance";
 export async function estimateDonation(
   token: TokenWithAmount,
   chainID: ChainID,
-  sender: string
+  sender: string,
+  tipAmount: number
 ): Promise<Exclude<EstimateStatus, "loading">> {
   try {
+    const totalAmount = +token.amount + tipAmount;
+
     const balance = await tokenBalance(token, chainID, sender);
-    if (balance < +token.amount) {
+    if (balance < totalAmount) {
```

Comment:
Addresses issue in PR https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2878/files - balance should be able to cover both donation amount *and* tip amount.